db/migrate/20190808084803_create_packages.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePackages < ActiveRecord::Migration[5.2]
+  def change
+    create_table :packages do |t|
+      t.string :package_name
+      t.string :company_name
```

Comment:
Think it makes sense to create companies table

Reply:
It is necessary for first demo?